File: python/mst/kruskal_test.py
from kruskal import minimum_spanning_forest
from mst_random_test_data import RandomTestData


def test_random_cases():
	for _ in range(10_000):
		td = RandomTestData()
		got = minimum_spanning_forest(td.n, td.edges.copy())
		td.assert_answer(got)

File: java/src/mst/KruskalTest.java
package mst;

import org.junit.Test;

public class KruskalTest {
	@Test
	public void testRandomCases() {
		for (int test = 0; test < 10_000; test++) {
			var td = new randomTestData();
			var got = Kruskal.minimumSpanningForest(td.n, td.edges.clone());
			td.assertAnswer(got);
		}
	}
}
